### ISSUE TITLE
Feature Request: Vibrate when Timer ends 

### ISSUE THREAD

**Issue by Asimos32** [FrSkyRC/ETHOS-Feedback-Community#246]
on Timer is it possible to have the follow Countdown Mode opion:
1.Mute
2. Beep
3. Speech
4. Vibrate
5. Beep + Vibrate
6.Speech+Vibrate
 or at least Vibrate

**Comment by centertrim** [comment]
Yep, I think in my ideal world the speech would be combined with a short vibration at 30 seconds remaining and then two longer vibrations when the timer hits zero.

**Comment by Asimos32** [comment]
CenterTrim. playing with Ethos i found out that is really easy do do that using by using Logig Switch and Special Function

**Comment by mawzthefinn** [comment]
Yes, this is achievable with Logic Switches & Special Functions, but some customizable haptics for the timers would be a nice addition

**Comment by mawzthefinn** [comment]
Will be in the next nightlies for 1.5.0 - see #2582 for images (look for the Haptic switch)
